src/components/Sidebar/components/NavItem/utils.js

```diff
@@ -25,4 +27,17 @@ const hasSelectedChild = children => {
   return false;
 };
 
-export default hasSelectedChild;
+const disabledIcon = <Lock />;
+
+export const getIcon = ({ defaultIcon, selected, selectedIcon, disabled }) => {
+  if (defaultIcon && disabled) {
```

Comment:
For easier understanding I'd create a separate variable here: 

```js
const hasIcon = !!defaultIcon;
```

Reply:
Is this what you meant?

https://github.com/sumup/circuit-ui/blob/82e376be05e6e1f988c5fca2d672828f1e46cff9/src/components/Sidebar/components/NavItem/utils.js#L32-L43

Reply:
Definitely clearer 👍

Reply:
Yes, that's what I had in mind :)